InternTrack.Core.Api/Services/Foundations/Interns/InternService.Validations.cs

```diff
@@ -107,6 +110,14 @@ private bool IsDateNotRecent(DateTimeOffset date)
             return timeDifference.Duration() > oneMinute;
         }
 
+        private void ValidateStorageIntern(Intern maybeIntern, Guid internId)
```

Comment:
Methods that are used outside of the file take precedence over methods that are used within the file.